L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigisPostLS1_cfi.py

```diff
@@ -1,5 +1,10 @@
 import FWCore.ParameterSet.Config as cms
 
+print " ######################################################################################################### "
+print " # WARNING: the module L1Trigger.CSCTriggerPrimitives.cscTriggerPrimitiveDigisPostLS1_cfi is deprecated. # "
```

Comment:
@slava77 You're right. The configuration of L1Trigger/CSCTriggerPrimitives needs a serious clean-up. I will do that next week.